torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -127,6 +128,60 @@ RegisterOperators reg(
            };
          },
          aliasAnalysisSpecialCase()),
+     Operator(
+         "prim::AllocateSlab() -> Storage",
+         [](const Node* node) -> Operation {
+           int64_t total_size = node->i(attr::total_size);
+           auto device_type =
+               static_cast<DeviceType>(node->i(attr::device_type));
+           return [total_size, device_type](Stack* stack) {
+             auto allocator = GetAllocator(device_type);
+             auto slab = c10::Storage(
+                 c10::Storage::use_byte_size_t(),
+                 total_size,
+                 allocator,
+                 /*resizable=*/false);
+             push(stack, std::move(slab));
+           };
+         },
+         aliasAnalysisSpecialCase()),
+     Operator(
+         prim::AllocateTensor,
+         [](const Node* node) -> Operation {
+           int64_t size = node->i(attr::size);
+           int64_t offset = node->i(attr::offset);
+           auto type = node->ty(attr::profiled_type)->expect<TensorType>();
+
+           return [offset, size, type](Stack* stack) {
+             c10::Storage slab;
+             pop(stack, slab);
+             uint8_t* start = static_cast<uint8_t*>(slab.data());
+             void* src = static_cast<void*>(start + offset);
```

Comment:
void* !! you love to see it

Reply:
yea i feel like a real pro now passing these around :)